app/jobs/pubsub_job.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class PubsubJob < ApplicationJob
+  retry_on(StandardError, wait: 5.minutes, attempts: 4)
+
+  self.queue_adapter = :pubsub
```

Comment:
This should be set in the environment files - you probably don't want this adapter for test environment.